.travis.yml

```diff
@@ -3,4 +3,6 @@ node_js:
   - "stable"
 services:
   - postgresql
+before_script:
```

Comment:
Hello fiyin
nice work 
I think you should fix a  new line here though
Thanks

Reply:
Fixed. Thanks